src/cryptography/x509.py

```diff
@@ -1344,3 +1344,41 @@ def extensions(self):
         """
         Returns an Extensions object containing a list of Revoked extensions.
         """
+
+
+class CertificateSigningRequestBuilder(object):
+    def __init__(self, subject_name=None, extensions=[]):
+        """
+        Creates an empty X.509 certificate request (v1).
+        """
+        self._subject_name = subject_name
+        self._extensions = extensions[:]
```

Comment:
Fixed in https://github.com/sigmavirus24/cryptography/commit/eece6d4ce8e50ed0f4869b062a5c77679e797b34